src/java/org/apache/cassandra/db/compression/CompressionDictionary.java

```diff
@@ -49,6 +49,14 @@ public interface CompressionDictionary extends AutoCloseable
      */
     byte[] rawDictionary();
 
+    /**
+     * Get memory occupied of this dictionary as a whole.
+     * Use for metrics exposing used memory in total.
+     *
+     * @return memory occuppied by this compression dictionary
+     */
+    int occupiedMemory();
```

Comment:
Looking at the method the next day... now I lean more towards adding `bytes` to the name. Maybe also add an explanation on return `int`, since we expect the dictionary size should be in KiBs.